### PR TITLE
Add ID translation for configure access multiple clusters page

### PR DESCRIPTION
**Pull request by adityaalifn**
add ID translation for `/docs/tasks/access-application-cluster/configure-access-multiple-clusters/`

part of this issue: https://github.com/kubernetes/website/issues/20111

/language id
/assign @kubernetes/sig-docs-id-reviews

### CODE REVIEW

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+- context:
+    cluster: development
+    namespace: ramp
+    user: developer
+  name: dev-ramp-up
+```
+
+Berkas konfigurasi diatas mendefinisikan sebuah konteks baru bernama `dev-ramp-up`.
+
+## Mengatur variabel lingkungan KUBECONFIG
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:257]: let's discuss if we should use "variabel lingkungan" or "variabel _environment_"
cc @kubernetes/sig-docs-id-reviews

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+
+### Linux
+```shell
+export KUBECONFIG_SAVED=$KUBECONFIG
+```
+### Windows PowerShell
+```shell
+$Env:KUBECONFIG_SAVED=$ENV:KUBECONFIG
+```
+Variabel lingkungan `KUBECONFIG` adalah sebuah daftar dari jalur-jalur (_paths_) menuju file konfigurasi.
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:271]:
```suggestion
Variabel lingkungan `KUBECONFIG` adalah sebuah daftar dari jalur-jalur (beragam _path_) menuju berkas konfigurasi.
```

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+$Env:KUBECONFIG=("config-demo;config-demo-2")
+```
+
+Di direktori `config-exercise` kamu, masukan perintah ini:
+
+```shell
+kubectl config view
+```
+
+Keluaran menunjukan informasi gabungan dari semua berkas yang terdaftar dalam variabel linkungan `KUBECONFIG` kamu.
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:293]: nit: missing "k" and typo
```suggestion
Keluaran menunjukkan informasi gabungan dari semua berkas yang terdaftar dalam variabel lingkungan `KUBECONFIG` kamu.
```

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 4]
```diff
@@ -0,0 +1,379 @@
...
+Secara khusus, perhatikan bahwa informasi gabungan tersebut memiliki konteks `dev-ramp-up` konteks dari berkas
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:294]: how about introducing comma here?
```suggestion
Secara khusus, perhatikan bahwa informasi gabungan tersebut memiliki konteks `dev-ramp-up`, konteks dari berkas 
```

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 5]
```diff
@@ -0,0 +1,379 @@
...
+
+Di direktori `config-exercise` kamu, masukan perintah ini:
+
+```shell
+kubectl config view
+```
+
+Keluaran menunjukan informasi gabungan dari semua berkas yang terdaftar dalam variabel linkungan `KUBECONFIG` kamu.
+Secara khusus, perhatikan bahwa informasi gabungan tersebut memiliki konteks `dev-ramp-up` konteks dari berkas 
+`config-demo-2` dan tiga konteks dari berkas `config-demo`:
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:295]:
```suggestion
`config-demo-2`, dan tiga konteks dari berkas `config-demo`:
```

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md [hunk 6]
```diff
@@ -0,0 +1,379 @@
...
+  name: exp-scratch
+```
+
+Untuk informasi lebih tentang bagaimana berkas Kubeconfig tergabung, lihat
+[Mengatur Akses Cluster Menggunakan Berkas Kubeconfig](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)
+
+## Jelajahi direktori $HOME/.kube
+
+Jika kamu sudah memiliki sebuah klaster, dan kamu bisa menggunakan `kubectl` untuk berinteraksi dengan
+klaster kamu, kemudian kamu mungkin memiliki sebuah file bernama `config` di
```
Comment (girikuncoro) [inline content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:327]:
```suggestion
klaster kamu, kemudian kamu mungkin memiliki sebuah berkas bernama `config` di
```

File: content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md (hunk 1 above)
Comment (adityaalifn) [reply content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:257]: In RSNI glosarium, `environment` is translated to `lingkungan`

Comment (girikuncoro) [reply content/id/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md:257]: make sense 👍

**Comment by girikuncoro** [comment]
@adityaalifn all good thanks for the changes, can you first squash the commits into 1 with good message first before getting this in?

/lgtm
/hold

**Comment by adityaalifn** [comment]
> @adityaalifn all good thanks for the changes, can you first squash the commits into 1 with good message first before getting this in?
> 
> /lgtm
> /hold

done mas @girikuncoro

**Pull request merged**